src/bundle/Resources/public/scss/fieldType/_ezformlabels.scss

```diff
@@ -0,0 +1,28 @@
+.ez-content-edit {
```

Comment:
During our conversation I didn't know how the CSS selectors will look like in the final version. Now I see that all the rules that begin with `.ez-content-edit` should be placed in the `_content-edit.scss` file.
Please, move all the rules to the file I mentioned above :) The same applies for the rule you've added in the `_base-field.scss`